staking_deposit/cli/exit_transaction_mnemonic.py

```diff
@@ -94,39 +111,56 @@ def exit_transaction_mnemonic(
-            credential.save_exit_transaction(validator_index=validator_index, epoch=epoch, folder=folder)
-            for credential, validator_index in items
-        ]
-
-    with click.progressbar(zip(transaction_filefolders, credentials),
-                           label=load_text(['msg_verify_exit_transaction']),
-                           show_percent=False,
-                           length=num_keys,
-                           show_pos=True) as items:
-        if not all(
-            verify_signed_exit_json(file_folder=file,
-                                    pubkey=credential.signing_pk.hex(),
-                                    chain_settings=credential.chain_setting)
-            for file, credential in items
-        ):
-            raise ValidationError(load_text(['err_verify_exit_transactions']))
+    credentials = []
+    with click.progressbar(length=num_keys, label=load_text(['msg_key_creation']),
+                           show_percent=False, show_pos=True) as bar:
+
+        executor_kwargs = [{
+            'mnemonic': mnemonic,
+            'mnemonic_password': mnemonic_password,
+            'index': index,
+            'amount': 0,
+            'chain_setting': chain_settings,
+            'hex_eth1_withdrawal_address': None,
+        } for index in key_indices]
+
+        with concurrent.futures.ProcessPoolExecutor() as executor:
+            for credential in executor.map(_credential_builder, executor_kwargs):
+                credentials.append(credential)
+                bar.update(1)
+
+    transaction_filefolders = []
+    with click.progressbar(length=num_keys, label=load_text(['msg_exit_transaction_creation']),
+                           show_percent=False, show_pos=True) as bar:
+
+        executor_kwargs = [{
+            'credential': credential,
+            'validator_index': validator_index,
+            'epoch': epoch,
+            'folder': folder,
+        } for credential, validator_index in zip(credentials, validator_indices)]
+
+        with concurrent.futures.ProcessPoolExecutor() as executor:
+            for filefolder in executor.map(_exit_exporter, executor_kwargs):
+                transaction_filefolders.append(filefolder)
+                bar.update(1)
+
+    all_valid_exits = True
+    with click.progressbar(length=num_keys, label=load_text(['msg_verify_exit_transaction']),
+                           show_percent=False, show_pos=True) as bar:
+
+        executor_kwargs = [{
+            'file_folder': file,
+            'credential': credential,
+        } for file, credential in zip(transaction_filefolders, credentials)]
+
+        with concurrent.futures.ProcessPoolExecutor() as executor:
+            for valid_exit in executor.map(_exit_verifier, executor_kwargs):
+                all_valid_exits &= valid_exit
+                bar.update(1)
+
+    if not all_valid_exits:
+        raise ValidationError(load_text(['err_verify_exit_transactions']))
```

Comment:
I noticed in `generate_bls_to_execution_change` we return upon the first credential mismatch. Is there a reason we continue to validate instead of aborting on first instance here? Something like:

```suggestion
    with click.progressbar(length=num_keys, label=load_text(['msg_verify_exit_transaction']),
                           show_percent=False, show_pos=True) as bar:

        executor_kwargs = [{
            'file_folder': file,
            'credential': credential,
        } for file, credential in zip(transaction_filefolders, credentials)]

        with concurrent.futures.ProcessPoolExecutor() as executor:
            for valid_exit in executor.map(_exit_verifier, executor_kwargs):
                bar.update(1)
                if not valid_exit:
                    raise ValidationError(load_text(['err_verify_exit_transactions']))
```